plugins/video-intelligence-monetization/frontend/public/javascripts/iframeResizer.min.js

```diff
@@ -0,0 +1,9 @@
+//# sourceMappingURL=iframeResizer.map
```

Comment:
I think this line messes up with concatenation/uglification process, can you remove it?